draft-ietf-tls-tls13.md

```diff
@@ -4195,6 +4195,17 @@ Cryptographic details:
   {{ffdhe-param}})?
 
 
+## Client Tracking Prevention
+
+Clients SHOULD NOT reuse a session ticket for multiple connections. Reuse
+of a session ticket allows passive observers to correlate different connections.
+Servers that issue session tickets SHOULD offer at least as many session tickets
+as the number of connections that a client might use; for example, a web browser
+using HTTP/1.1 might open six connections to a server. Servers SHOULD issue new
```

Comment:
If you mention HTTP/1.1, then probably it needs a informative reference to the HTTP/1.1 spec (maybe [RFC 7231](https://tools.ietf.org/html/rfc7231)).